src/boot/efi/boot.c

```diff
@@ -381,91 +385,102 @@ static VOID print_status(Config *config, CHAR16 *loaded_image_path) {
 
         console_query_mode(&x_max, &y_max);
 
-        Print(L"systemd-boot version:   " GIT_VERSION "\n");
-        Print(L"architecture:           " EFI_MACHINE_TYPE_NAME "\n");
-        Print(L"loaded image:           %s\n", loaded_image_path);
-        Print(L"UEFI specification:     %d.%02d\n", ST->Hdr.Revision >> 16, ST->Hdr.Revision & 0xffff);
-        Print(L"firmware vendor:        %s\n", ST->FirmwareVendor);
-        Print(L"firmware version:       %d.%02d\n", ST->FirmwareRevision >> 16, ST->FirmwareRevision & 0xffff);
-        Print(L"console size:           %d x %d\n", x_max, y_max);
-        Print(L"Console mode:           %d/%d\n", ST->ConOut->Mode->Mode, ST->ConOut->Mode->MaxMode - 1);
-        Print(L"SecureBoot:             %s\n", yes_no(secure_boot_enabled()));
-
-        if (efivar_get_boolean_u8(EFI_GLOBAL_GUID, L"SetupMode", &modevar) == EFI_SUCCESS)
-                Print(L"SetupMode:              %s\n", modevar ? L"setup" : L"user");
-
-        if (shim_loaded())
-                Print(L"Shim:                   present\n");
-
-        if (efivar_get_uint64_le(EFI_GLOBAL_GUID, L"OsIndicationsSupported", &indvar) == EFI_SUCCESS)
-                Print(L"OsIndicationsSupported: %d\n", indvar);
-
-        Print(L"\n--- press key ---\n\n");
+        #define line(property, format, ...) ({ \
+                if (format) { \
+                        Print(L"%*s: ", padding, property); \
+                        Print(format, ## __VA_ARGS__); \
+                        Print(L"\n"); \
+                } \
+        })
```

Comment:
define this above the function, after all there's no concept of "local macros".

and please uppercase the name, we usually mark macros that way. (we only use non-uppercase in case the macros are very function like, but this one is not)